src/Inkluzitron/Services/PointsService.cs

```diff
@@ -446,6 +446,29 @@ private async Task<MagickImage> RenderPointsBaseFrameAsync(int position, IUser u
 
             await DrawGraphOnPointsImage(image, user, 250, 60);
 
+            using var dbContext = DatabaseFactory.Create();
+            var userDb = await dbContext.Users.AsQueryable().Include(u => u.Badges)
```

Comment:
Když už sis na to dělal extension metodu v uživatelské službě, tak jsi ji mohl použít.

Reply:
To byla extension metoda v concurrency bazmeku, takže all good.